.travis.yml

```diff
@@ -1,24 +1,27 @@
 language: node_js
 sudo: false
 node_js:
-- '8'
-- '10'
+  - '8'
+  - '10'
+install:
+  # Node.js 8 comes with an npm that doesn't support "npm ci"
+  - if [[ "$TRAVIS_NODE_VERSION" == "8" ]]; then npm install -g npm@^5.7.1; fi
```

Comment:
Do you want to install an exact version number, or latest `npm@5`?

Reply:
Latest `npm@5`, or really, any version that's at least `5.7.1`

Reply:
Neat 👍